app/assets/stylesheets/geoblacklight.scss

```diff
@@ -1,3 +1,7 @@
 @import 'bootstrap/scss/bootstrap';
 @import 'blacklight-frontend/app/assets/stylesheets/blacklight/blacklight';
-@import '@geoblacklight/frontend/app/assets/stylesheets/geoblacklight/geoblacklight';
+@import '@geoblacklight/frontend/dist/stylesheets/geoblacklight/geoblacklight';
+
+.palo-alto-base {
+  background-color: #175E54!important;
```

Comment:
Why do we need `!important`?

Reply:
Yup. Bootstrap overrides the background color on the header.

Reply:
In argo we are doing this instead:
```
.stone-dark {
  --bs-dark-rgb: var(--stanford-stone-dark-rgb);
}
```

I think you could do the same here (but for palo-alto)

Reply:
That makes sense. Thanks. It worked.